### PR TITLE
Feature/SOF-7469 clean 

### PR DESCRIPTION
**Pull request by VsevolodX**

### CODE REVIEW

**Comment by VsevolodX** [comment]
GIT LFS seems to be broken. LFS shows it tracks and adds the file, but it comes to GitHub as a plain one, not via LFS. 
```
(.venv) 12:19:38 GREEN ➜  documentation git:(feature/SOF-7469-clean) ✗ gs                                                                                          
On branch feature/SOF-7469-clean
Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
        new file:   images/tutorials/materials/defects/defect_creation_point_substitution_graphene/0.png

(.venv) 12:19:39 GREEN ➜  documentation git:(feature/SOF-7469-clean) ✗ git lfs status                                                                              
On branch feature/SOF-7469-clean

Objects to be committed:

        images/tutorials/materials/defects/defect_creation_point_substitution_graphene/0.png (LFS: 1cd75fd)

Objects not staged for commit:


(.venv) 12:19:48 GREEN ➜  documentation git:(feature/SOF-7469-clean) ✗ git commit -m "update: add image with git lfs 2"
[feature/SOF-7469-clean 2a838478] update: add image with git lfs 2
 1 file changed, 3 insertions(+)
 create mode 100644 images/tutorials/materials/defects/defect_creation_point_substitution_graphene/0.png
```

**Comment by VsevolodX** [comment]
git lfs is installed, it tracks and adds png files.

```
(.venv) 18:12:01 GREEN ➜  documentation git:(feature/SOF-7469-clean) ✗ git lfs status 
On branch feature/SOF-7469-clean

Objects to be committed:


Objects not staged for commit:

        images/tutorials/materials/defects/defect_creation_point_substitution_graphene/0.png (LFS: 1cd75fd -> File: deleted)

(.venv) 18:12:10 GREEN ➜  documentation git:(feature/SOF-7469-clean) ✗ git add images/tutorials                                                                            
(.venv) 18:12:35 GREEN ➜  documentation git:(feature/SOF-7469-clean) ✗ gs                      
On branch feature/SOF-7469-clean
Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
        renamed:    images/tutorials/materials/defects/defect_creation_point_substitution_graphene/0.png -> images/tutorials/materials/defects/defect_creation_point_substitution_graphene/0-paper-image.png
        new file:   images/tutorials/materials/defects/defect_creation_point_substitution_graphene/1-standata-graphene.png
        new file:   images/tutorials/materials/defects/defect_creation_point_substitution_graphene/2-advanced-supercell.png
        new file:   images/tutorials/materials/defects/defect_creation_point_substitution_graphene/3-wave-graphene-supercell.png
        new file:   images/tutorials/materials/defects/defect_creation_point_substitution_graphene/4-threejs-editor-coordinates.png
        new file:   images/tutorials/materials/defects/defect_creation_point_substitution_graphene/5-jl-setup.png
        new file:   images/tutorials/materials/defects/defect_creation_point_substitution_graphene/6-jl-result-preview.png
        new file:   images/tutorials/materials/defects/defect_creation_point_substitution_graphene/7-wave-result.png

(.venv) 18:12:36 GREEN ➜  documentation git:(feature/SOF-7469-clean) ✗ git lfs status          
On branch feature/SOF-7469-clean

Objects to be committed:

        images/tutorials/materials/defects/defect_creation_point_substitution_graphene/0.png -> images/tutorials/materials/defects/defect_creation_point_substitution_graphene/0-paper-image.png (LFS: 1cd75fd -> LFS: 1cd75fd)
        images/tutorials/materials/defects/defect_creation_point_substitution_graphene/1-standata-graphene.png (LFS: b2f2b1d)
        images/tutorials/materials/defects/defect_creation_point_substitution_graphene/2-advanced-supercell.png (LFS: 2cb0586)
        images/tutorials/materials/defects/defect_creation_point_substitution_graphene/3-wave-graphene-supercell.png (LFS: 0afc480)
        images/tutorials/materials/defects/defect_creation_point_substitution_graphene/4-threejs-editor-coordinates.png (LFS: 282d8ed)
        images/tutorials/materials/defects/defect_creation_point_substitution_graphene/5-jl-setup.png (LFS: 240a2f9)
        images/tutorials/materials/defects/defect_creation_point_substitution_graphene/6-jl-result-preview.png (LFS: 7fce8ef)
        images/tutorials/materials/defects/defect_creation_point_substitution_graphene/7-wave-result.png (LFS: a15c5df)

Objects not staged for commit:


(.venv) 18:12:46 GREEN ➜  documentation git:(feature/SOF-7469-clean) ✗ 
```

Next commit will push them supposedly with LFS, if that is not the case, Git LFS broken for this PR/repo

File: lang/en/docs/tutorials/materials/specific/defect-creation-point-substitution-graphene.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+
+## Interactive JupyterLite Notebook
+
+The following JupyterLite notebook demonstrates the process of creating materials with substitution defects in graphene.
+
+
+{% with origin_url=config.extra.jupyterlite.origin_url %}
+{% with notebooks_path_root=config.extra.jupyterlite.notebooks_path_root %}
+{% with notebook_name='create_point_defect.ipynb' %}
```
Comment (timurbazhirov) [inline lang/en/docs/tutorials/materials/specific/defect-creation-point-substitution-graphene.md:130]: To be updated after jupyterlite is merged

File: .gitignore [hunk 2]
```diff
@@ -15,3 +15,6 @@ service-account-key.json
 # ignore media files in scripts folder
 scripts/*.mp3
 scripts/*.mp4
+
+# Pyenv
+.python-version
```
Comment (timurbazhirov) [inline .gitignore:20]: File should be called 01-figure-from-manuscript.png

File: .gitignore [hunk 3]
```diff
@@ -15,3 +15,6 @@ service-account-key.json
 # ignore media files in scripts folder
 scripts/*.mp3
 scripts/*.mp4
+
```
Comment (timurbazhirov) [inline .gitignore:18]: Jupyterlite section should have:
- Overview: explaining the basics of the environment and listing the content of the section
- Actions:
-- Open
-- Run
-- Upload File
-- Copy Notebook

File: lang/en/docs/tutorials/materials/specific/defect-creation-point-substitution-graphene.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
+---
+# YAML header
+render_macros: true
```
Comment (timurbazhirov) [inline lang/en/docs/tutorials/materials/specific/defect-creation-point-substitution-graphene.md:3]: Remove

Comment (VsevolodX) [reply lang/en/docs/tutorials/materials/specific/defect-creation-point-substitution-graphene.md:3]: Seems to be required

File: .gitignore (hunk 3 above)
Comment (VsevolodX) [reply .gitignore:18]: Did something along those lines

File: lang/en/docs/jupyterlite/overview.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# Introduction to JupyterLite
+
+The [**JupyterLite**](https://jupyterlite.readthedocs.io/en/stable/) environment is a lightweight implementation of JupyterLab that runs entirely in the web browser. Without the need for setup and installation, user can open, edit and run Jupyter notebooks.
```
Comment (timurbazhirov) [inline lang/en/docs/jupyterlite/overview.md:3]: Let's embed it here too for demonstration purposes

File: lang/en/docs/jupyterlite/file-storage-synchronization.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# File Storage and Synchronization in JupyterLite
```
Comment (timurbazhirov) [inline lang/en/docs/jupyterlite/file-storage-synchronization.md:1]: JupyterLite: File Storage and Synchronization

File: lang/en/docs/jupyterlite/accessing-jupyterlite.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
+# Accessing JupyterLite
```
Comment (timurbazhirov) [inline lang/en/docs/jupyterlite/accessing-jupyterlite.md:1]: JupyterLite: Accessing the Environment.

File: lang/en/docs/jupyterlite/accessing-jupyterlite.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+
+One can access JupyterLite from various points in the Mat3ra ecosystem. Below are the available options:
+
+### 1. Materials Designer
```
Comment (timurbazhirov) [inline lang/en/docs/jupyterlite/accessing-jupyterlite.md:5]: Should be H2

File: lang/en/docs/jupyterlite/actions.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
+# Using JupyterLite and Data Access
```
Comment (timurbazhirov) [inline lang/en/docs/jupyterlite/actions.md:1]: Jupyterlite: Common Actions

File: lang/en/docs/jupyterlite/actions.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+## Actions in JupyterLite
+
+Within the JupyterLite environment, the following actions are available:
+
+- **Open**: Open an existing notebook or file.
+- **Run**: Execute code cells within the notebook.
+- **Upload File**: Upload files from your local system.
+- **Copy Notebook**: Create a copy of an existing notebook.
+
+### Open
```
Comment (timurbazhirov) [inline lang/en/docs/jupyterlite/actions.md:12]: Should be H2

File: lang/en/docs/jupyterlite/data-transfer.md [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+
+The first parameter specifies the name of the global variable (`"materials_in"`) where the received data will be stored.
+The second parameter, `globals()`, ensures that the function operates correctly across both Pyodide and Python environments. It allows `get_data` to dynamically interact with the global namespace of the script.
+
+Data Handling:
+
+The materials data is initially stored in a global variable named `data_from_host`, which is updated in response to changes in material selection or the materials themselves.
+In the context of the Pyodide environment, `data_from_host` becomes available after the Pyodide kernel has loaded and the extension set the data.
+
+### Send Materials Back to Materials Designer
```
Comment (timurbazhirov) [inline lang/en/docs/jupyterlite/data-transfer.md:22]: Should be H2

File: lang/en/docs/jupyterlite/data-transfer.md [hunk 12]
```diff
@@ -0,0 +1,35 @@
+# Accessing Data from the Platform in JupyterLite
+
+To work with materials data inside the JupyterLite environment launched from Materials Designer, use the following code snippet:
```
Comment (timurbazhirov) [inline lang/en/docs/jupyterlite/data-transfer.md:3]: Should have H2 - Overview or something like that

File: lang/en/docs/jupyterlite/overview.md [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+## [Accessing JupyterLite](./accessing-jupyterlite.md)
+Provides a detailed guide on how to access JupyterLite via the **Materials Designer**, **Mat3ra Platform**, or a direct URL, with accompanying visuals.
+
+## [Data Transfer](./data-transfer.md)
+Explains how to transfer data between JupyterLite and Materials Designer, including code snippets for importing and exporting materials data.
+
+## [File Storage and Synchronization](./file-storage-synchronization.md)
+Describes how JupyterLite stores files locally in the browser, discusses synchronization limitations across devices, and outlines steps for clearing the local cache when updates are available.
+
+## [Actions](./actions.md)
```
Comment (timurbazhirov) [inline lang/en/docs/jupyterlite/overview.md:20]: Add links to original JupyterLite documentation.

We should also add links to JupyterLite basic documentation

**Comment by pranabdas** [comment]
Something has changed on the netlify side, https://github.com/Exabyte-io/documentation/pull/280/commits/8207ca1e56f8afc8a494382d2b4a1f6ddeb02321 fixes the deploy issue for us.

**Comment by timurbazhirov** [comment]
Superseded by #285

**Pull request closed without merging**